pkg/internal/checkup/checkup_test.go

```diff
@@ -288,6 +288,20 @@ func TestRunFailure(t *testing.T) {
 			},
 			expectedRunErr: fmt.Errorf(packetsDontMatchErrMsg, trafficGenSentPackets, vmUnderTestReceivedPackets),
 		},
+		{
+			description: "fail because for multiple error cases",
+			results: status.Results{
+				TrafficGenSentPackets:        0,
+				TrafficGenOutputErrorPackets: trafficGenOutputErrPackets,
+				TrafficGenInputErrorPackets:  trafficGenInputErrPackets,
+				VMUnderTestTxDroppedPackets:  vmUnderTestTxDroppedPackets,
+				VMUnderTestRxDroppedPackets:  vmUnderTestRxDroppedPackets,
+				VMUnderTestReceivedPackets:   0,
+			},
+			expectedRunErr: fmt.Errorf("%s, %s",
```

Comment:
If the traffic gen had sent zero packets, why are the following values non-zero?
```
TrafficGenOutputErrorPackets: trafficGenOutputErrPackets,
TrafficGenInputErrorPackets:  trafficGenInputErrPackets,
VMUnderTestTxDroppedPackets:  vmUnderTestTxDroppedPackets,
VMUnderTestRxDroppedPackets:  vmUnderTestRxDroppedPackets,
```				
Furthermore, why does the "no packets were sent from the traffic generator" error is not expected?

Reply:
there was a problem with the assertion. I fixed it in the new push